### ISSUE TITLE
no-invalid-html doesn't always warn on invalid HTML

### ISSUE THREAD

**Issue by julienw** [43081j/eslint-plugin-lit#243]
Something like that isn't errored:

```js
class MyElement extends LitElement {
  render() {
    return html`
      </div>
    `;
  }
}
```

That is, if there is a closing tag for an element that wasn't opened.
These ones can be difficult to spot, and this can lead to other tools not working properly (see https://github.com/prettier/prettier/issues/19051 for example).
